cluster_library.c

```diff
@@ -1870,7 +1870,11 @@ PHP_REDIS_API void cluster_variant_resp(INTERNAL_FUNCTION_PARAMETERS, redisClust
                 RETVAL_TRUE;
                 break;
             case TYPE_BULK:
-                RETVAL_STRINGL(r->str, r->len);
+                if (r->len > -1) {
```

Comment:
I think it would be cleaner if check looks like `if (r->len < 0) {` instead of `if (r->len > -1) {`